docs/source/documentation.rst

```diff
@@ -14,6 +14,7 @@ The following sections will help you install all the tools you need to run AxonD
            However, please note that our continuous integration testing framework (TravisCI) only tests AxonDeepSeg
            for Unix-style systems, so releases may be more unstable for Windows than Linux/macOS.
 
+In case, you find trouble installing ADS or FSLeyes pulugin you could refer to our youtube video 
```

Comment:
not the right place for it-- also there is a typo "pulugin"

the thing to do is to embed the video in the FSLeyes section

Reply:
Sure, will add instructions in the FSLeyes section and will correct typo. I'll ask for review once done because I'm thinking of adding a youtube logo as well in the readme and RTD, which is not done yet.

Reply:
> I'll ask for review once done because I'm thinking of adding a youtube logo as well in the readme and RTD, which is not done yet.

no need for the youtube logo-- just embed the stuff

Reply:
exampel: https://spine-generic.readthedocs.io/en/latest/analysis-pipeline.html#segmentation-and-vertebral-labeling